test/libethereum/Block.cpp

```diff
@@ -50,6 +50,8 @@ BOOST_AUTO_TEST_CASE(bStructures)
 
 BOOST_AUTO_TEST_CASE(bStates)
 {
+	if (!dev::test::Options::get().quadratic)
```

Comment:
I don't know much about how Boost.Test operate, but couldn't we do in a more dynamic way, like having external config files with a list of tests to run ?
So that we won't need to recompile tests every time we move tests between fast/slow suites ?